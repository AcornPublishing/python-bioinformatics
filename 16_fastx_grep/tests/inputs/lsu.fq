@M00773:480:000000000-BLYPT:1:2106:12063:1841 1:N:0:AGGCGACCTTA
TTTCTGTGCCAGCAGCCGCGGTAAGACAGAGGTGGCGAGCGTTGTTCGGATTTACTGGGCGTAAAGCGCGGGTAGGCGGTTCGGCCAGTCAGATGTGAAATCCCACCGCTTAACGGTGGAACGGCGTCTGATACTACCGGACTTGAGTGCAGGAGAGGAGGGTGGAATTTCCGGTGTAGCGGTGAAATGCGTAGAGATCGGAAGGAACACCAGTGGCGAAGGCGGCCCTCTGGACTGCAACTGACGCTGAGACGCGAAAGCGTGGGGAGCACACAGGATTAGATACCCTGGTAGTCAACGC
+
CCCCCGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGEFGGFEGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGFGGGGGGGGGGGGGGGGGGEGGGGGGGEGGGGGGDGDGGGGGGGGGGGFDGGGGGGGGFFFFDFG7FFGGGGGGGGG7EGGGGGDGGEGGGGGG=EFGDGGFGGDEGGGGFFC5;EEDFEFGEGFCFGEECC8?5CEE*:5*;?FGGFGCCFGAFFGGGDGGFFGCDECGGGGE;EE8EC=390;575>8<+9FGGFC<8CGFF:9+9,<D5)
@ITSLSUmock2p.ITS_M01380:138:000000000-C9GKM:1:1101:14440:2042 2:N:0
CAAGTTACTTCCTCTAAATGACCAAGCCTAGTGTAGAACCATGTCGTCAGTGTCAGTCTGAGTGTAGATCTCGGTGGTCGCCGTATCATTAAAAAAAAAAATGTAATACTACTAGTAATTATTAATATTATAATTTTGTCTATTAGCATCTTATTATAGATAGAAGATATTATTCATATTTCACTATCTTATACTGATATCAGCTTTATCAGATCACACTCTAGTGAAGATTGTTCTTAACTGAAATTTCCTTCTTCATACAGACACATTAATCTTACCTA
+
EFGGGGGGGGGCGGGGGFCFFFGGGGGFGGGGGGGGGGGFGGGGGGGFGFFFCFGGFFGGGGGGGGGFGGGGFGGGDG<FD@4@CFFGGGGCFFAFEFEG+,9,,,,99,,,5,,49,4,8,4,444,4,4,,,,,,,,,,,,,,8,,,,63,,,,,,,,376,3,,,,,,,8,,,,,,,,,+++++++++++++3++25+++0+*+0+*0+***))*0))1/+++**************.****.*******0*********/(,(/).)))1)).).).
@M00773:480:000000000-BLYPT:1:2106:12063:1841 2:N:0:AGGCGACCTTA
ACCCGCCAATTTCTTTGAGTTTCAACCTTGCGGCCGTACTCCCCAGGCGGGGCACTTACTCCGTTAGGTGCGGCACGGGAGGGGTCGATACCTCCCACACTTAGTGCCCATCGTTTAGGGCGTGGACTACCAGGGTATCTAATCCTGTTTGCTCCCCACGCTTTCGCGTCTCAGCGTCAGTTGCAGTCCAGAGGGCCGCCTTCGCCACTGGTGTTCCTTCCGAGCCCTACGCATTTCACCGCTACACCGGAAATTCCACCCTCCTCTCCTGCACTCAAGTCTGGCAGTAGCAGACGCCGTT
+
CCCCCDGGGGGGGGGGGGG<CECGGGFGGGGGGGGGGGGGF@FGGGGGGFDGGGGCDFGGGGGGCGGGGFDGCFFGGCGEDGEF:FGGGGCGCFGGFFGG@EGGGGGGDGFGGCF7D>FGGG><FCEGCFEFGEE@<:CFGGGGGCGGCGFGEC<<EGG:C:E5FE*:?E8C?ECEGG88C;E9AFFFGF@:EF*;88*/A5EFG8*/ACC**:*7<00C:ED))(-7>)0)9DEDFFF>8=*0.6C75C>DD?:@*)9(41:??FFF(9:2<FF70?):A).3)(,46))5)..,3(,41
@ITSLSUmock2p.ITS_M01384:138:000000000-C9GKM:1:1101:14440:2043 2:N:0
ACCCGTCAATTTCTTTAAGTTTTAGCCTTGCGACCGTACTCCCCAGGCGGTGCACTTAGTGGTTTTCCGGCGACCCGGGCGGCGTCAGAGCCCCCCAAGTCTCGTGCACATCGTTTACGGCGTGGACTACCAGGGTATCTAATCCTGTTTGATCCCCACGCTTTCGTGCCTCAGCGTCAGTACCGGCCCAGCCACCCGTCTTCACCTTCGGCGTTCCTGTAGATATCTACGCATTTCACCGCTACACCTACAGTTCCGGTGGCGCCTACCGGCCTCAAGAAACGCAGTATGCCCAGCTATT
+
CCCCCGGGGGGGGGGGGGGGGGGGGGGGGEGGGGGGGGGGGGGGFGGGGGGGGGGGG9FGGGGGGGGGGGGGGGGGGGGGGGGGGGGEGGGGGGGGGCGFGFGGGGGGGGGGGGGGGGGDGGGDEEFGGGGGGGGGGGGFFGGG9EGFGGFCGGCGGGCCGEGGGEGFGGGG6B6*=EGGGGGGG8ECCEGGGGCFGDEEGGG?FGFGGC39:>EGGGGGGF7=9:A@FBF>DGCGFF=75C=DBCF74DFFFF*/91B>9>?9?>>:B?>F>FBB:??200:699>?AA2)34F?2))54
