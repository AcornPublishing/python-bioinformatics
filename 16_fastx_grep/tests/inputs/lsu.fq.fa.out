>ITSLSUmock2p.ITS_M01380:138:000000000-C9GKM:1:1101:14440:2042 2:N:0
CAAGTTACTTCCTCTAAATGACCAAGCCTAGTGTAGAACCATGTCGTCAGTGTCAGTCTG
AGTGTAGATCTCGGTGGTCGCCGTATCATTAAAAAAAAAAATGTAATACTACTAGTAATT
ATTAATATTATAATTTTGTCTATTAGCATCTTATTATAGATAGAAGATATTATTCATATT
TCACTATCTTATACTGATATCAGCTTTATCAGATCACACTCTAGTGAAGATTGTTCTTAA
CTGAAATTTCCTTCTTCATACAGACACATTAATCTTACCTA
>ITSLSUmock2p.ITS_M01384:138:000000000-C9GKM:1:1101:14440:2043 2:N:0
ACCCGTCAATTTCTTTAAGTTTTAGCCTTGCGACCGTACTCCCCAGGCGGTGCACTTAGT
GGTTTTCCGGCGACCCGGGCGGCGTCAGAGCCCCCCAAGTCTCGTGCACATCGTTTACGG
CGTGGACTACCAGGGTATCTAATCCTGTTTGATCCCCACGCTTTCGTGCCTCAGCGTCAG
TACCGGCCCAGCCACCCGTCTTCACCTTCGGCGTTCCTGTAGATATCTACGCATTTCACC
GCTACACCTACAGTTCCGGTGGCGCCTACCGGCCTCAAGAAACGCAGTATGCCCAGCTAT
T
